tests/test_entities.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pytest
+
```

Comment:
Why the extra space? Thought you didn't like it.

Reply:
Only for `import foo` vs `from foo import bar`, but we can define the standard or just use PEP8 which is what you did originally I believe